login-workflow/src/screens/InviteRegistrationPager.tsx

```diff
@@ -309,7 +321,9 @@ export const InviteRegistrationPager: React.FC<InviteRegistrationPagerProps> = (
             {
                 name: 'Complete',
                 pageTitle: t('REGISTRATION.STEPS.COMPLETE'),
-                pageBody: (
+                pageBody: injectedUIContext.customRegistrationSuccessScreen ? (
+                    (getCustomRegistrationSuccessScreen() as JSX.Element)
+                ) : (
```

Comment:
You could probably simplify this a bit and save yourself the extra function (i would define the customSuccess variable towards the top to give you shorthand that's a little more readable too:
```
const customSuccess = injectedUIContext.customRegistrationSuccessScreen;
```
```suggestion
                pageBody: customSuccess ? (
                    <>
                        {typeof customSuccess === 'function' && customSuccess(navigation)}
                        {typeof customSuccess !== 'function' && customSuccess}
                    </>
                ) : (
```